src/NuGet.Services.Validation/Entities/CvsScanOperationState.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NuGet.Services.Validation.Entities
+{
+    /// <summary>
+    /// Status of the cvs scan operations
+    /// </summary>
+    public class CvsScanOperationState
+    {
+
+        /// <summary>
+        /// The database-mastered identifier for this operation.
+        /// </summary>
+        public long Key { get; set; }
+
+        /// <summary>
+        /// Validation ID for which this operation is performed.
+        /// </summary>
+        public Guid ValidationSetId { get; set; }
+
+        /// <summary>
+        /// Status of the operation performed.
+        /// </summary>
+        public int Status { get; set; }
+
+        /// <summary>
+        /// Job Id of the operation.
```

Comment:
Could we clarify this is the CVS related?

```suggestion
        /// CVS's ID for the content scan job.
```